eq-author-api/schema/resolvers/pages/questionPage.js

```diff
@@ -89,12 +94,16 @@ Resolvers.Mutation = {
       typeof position === "number" ? position : section.pages.length;
     section.pages.splice(insertionPosition, 0, page);
     await saveQuestionnaire(ctx.questionnaire);
+
+    ctx.validationErrorInfo = await validateQuestionnaire(ctx.questionnaire);
```

Comment:
Happy to leave like this for this PR but we should probably look at moving the `saveQuestionnaire` and `validateQuestionnaire` logic to some point outside of the mutations.